index.html

```diff
@@ -2,237 +2,146 @@
+            </div>
+            <input class="alignment-start button" type="button" value="Get My Resume">
+        </div>
+        <div class="languages-section">
+            <div class="list-heading">
+                <h3>Languages</h3>
+                <i class="fas fa-chevron-down fa-xs"></i>
+            </div>
+            <ul>
+                <li>
+                    <img src="images/javaScript.svg" alt="JS icon">
+                    <p>Java Script</p>
+                </li>
+                <li>
+                    <img src="images/html.svg" alt="HTML icon">
+                    <p>HTML</p>
+                </li>
+                <li>
+                    <img src="images/css.svg" alt="CSS icon">
+                    <p>CSS</p>
+                </li>
+            </ul>
+            <div class="list-heading">
+                <h3>Frameworks</h3>
+                <i class="fas fa-chevron-right fa-xs"></i>
+            </div>
+            <div class="list-heading">
+                <h3>Skills</h3>
+                <i class="fas fa-chevron-right fa-xs"></i>
+            </div>
+        </div>
+    </section>
+
+    <div class="white-background">
+        <section class="contact-section" id="contact">
+            <div class="width-constraint-container">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a
+                    project that needs coding. I’d love to help with it.</p>
+                <form action="https://formspree.io/f/xrgjbnyl" method="post" class="contact-form">
+                    <input type="text" name="name" placeholder="Your Name" maxlength="30" required>
+                    <input type="email" name="email" placeholder="Your Email" required>
+                    <textarea name="message" placeholder="Write your message here" maxlength="500" rows="7"
+                              required></textarea>
+                    <input type="submit" class="button" value="Get In Touch">
+                </form>
+            </div>
+        </section>
     </div>
-  </div>
-</section>
 
-<div class="white-background">
-  <section class="contact-section" id="contact">
-    <div class="width-constraint-container">
-      <h2>Contact me</h2>
-      <p>If you have an application you are interested in developing, a feature that you need built or a
-        project that needs coding. I’d love to help with it.</p>
-      <form action="https://formspree.io/f/xrgjbnyl" method="post" class="contact-form">
-        <input type="text" name="name" placeholder="Your Name" maxlength="30" required>
-        <input type="email" name="email" placeholder="Your Email" required>
-        <textarea name="message" placeholder="Write your message here" maxlength="500" rows="7"
-                  required></textarea>
-        <input type="submit" class="button" value="Get In Touch">
-      </form>
+    <div id="popup-window">
```

Comment:
- [ ] OPTIONAL: I would strongly recommend you to add overflow: hidden on body when the `popup-window` is visible. This way, users will not  be able to scroll the page.

Reply:
@ShoiraTa  Yes I realized that but I kinda looked at it as a feature lol, but I will remove this. I can see it looks a bit weird.